ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/termsandconditions/v1/termsandconditions/_cq_design_dialog/.content.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:granite="http://www.adobe.com/jcr/granite/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+  jcr:primaryType="nt:unstructured"
+  jcr:title="Adaptive Form Container"
+  sling:resourceType="cq/gui/components/authoring/dialog"
+  trackingFeature="core-components:design-dialog:adaptiveform-termsandconditions:v1">
+    <content
+      jcr:primaryType="nt:unstructured"
+      sling:resourceType="granite/ui/components/coral/foundation/container">
+        <items jcr:primaryType="nt:unstructured">
+            <tabs
+              jcr:primaryType="nt:unstructured"
+              sling:resourceType="granite/ui/components/coral/foundation/tabs">
+                <items jcr:primaryType="nt:unstructured">
+                    <assetmapping
+                      jcr:primaryType="nt:unstructured"
+                      jcr:title="Default Components"
+                      sling:resourceType="granite/ui/components/coral/foundation/container"
+                      margin="{Boolean}true"
+                      granite:hide="{Boolean}true" />
+                    <responsivesettings
+                      jcr:primaryType="nt:unstructured"
+                      jcr:title="Responsive Settings"
+                      sling:resourceType="granite/ui/components/coral/foundation/container"
+                      margin="{Boolean}true"
+                      granite:hide="{Boolean}true" />
+                    <styletab
```

Comment:
This tab should also be available for enacapsulated components such as link, checkbox...